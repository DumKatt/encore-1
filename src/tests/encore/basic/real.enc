class Main
  def main() : void
    let pi = 3.141593 in
      {print "Pi is something like ";
       print pi}
