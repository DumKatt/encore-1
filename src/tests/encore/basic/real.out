Pi is something like 
3.141593
